### PR TITLE
Week 2 Submission

### PR DESCRIPTION
**Pull request by mcardoso3-stat6250**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,42 @@
-# Course Questions Wiki
+[Course Textbook Chapter 1, Problem 1]
+- Question (mcardoso3~stat6250): How many observations and variables are needed for a large enough sample size to draw a strong conclusion from the data provided?
```
Comment (prof-lankham) [inline README.md]: Throughout, you appear to be using a tilde (~) in your Slack username, rather than a dash (-); please update to use a dash.

**Review by prof-lankham — CHANGES_REQUESTED** [review]
Please address comments before this PR can be accepted.

**Comment by mcardoso3-stat6250** [comment]
Thank you, Quiz 2 has been updated.

**Review by prof-lankham — CHANGES_REQUESTED** [review]
Your username now appears to be correct throughout. However, it appears that you are editing the README.md file at the root of the repo, rather than week-02/README.md. Please move your contributions to Week 2's README.md file, and please interlace your contributions with existing contributions so the all questions/answers for a give Quiz Problem/SAS Recipe appear under a common header.

**Comment by mcardoso3-stat6250** [comment]
Ok, I believe I updated week 2, but I'm not entirely sure.  If not, please let me know.

**Comment by prof-lankham** [comment]
You can see a summary of all changed in the PR as follows: https://github.com/stat6250/course-questions-wiki/pull/48/files

Note that the file being changed is "README.md", not "week-02/README.md"

**Comment by prof-lankham** [comment]
Your most recent commit still has edits to the README.md file at the root-level of the repo, rather than the README.md file in the week-02 folder.

Please message me in Slack if it would be helpful to talk through any of the details of submission process.

**Comment by prof-lankham** [comment]
This is being closed in order to prevent it from becoming stale.

Please message me in Slack to discuss further.

**Pull request closed without merging**